### DISCUSSION TITLE
[Mumble] Same client ip when using a proxy hit the 32 default limit

### DISCUSSION THREAD

**Discussion by joelwurtz** [citizenfx/fivem#1599] [Engineering Discussion]
Since https://github.com/citizenfx/fivem/commit/04af423c8496be1c0e2872a1ee850fd114eda5d5

When using mumble and having a tcp proxy in front of the fivem server, the client ip address is always the one from the tcp proxy (when using a configuration similar to https://docs.fivem.net/docs/server-manual/proxy-setup/)

So after reaching 32 players new players cannot connect to mumble.

I'm not sure if we can get the real ip address (i think nginx create a connection and copy input stream to output stream, it's not a passthrough)

The current work around is to upgrade the max client per ip limit with the convar however i'm not sure if it's a desired behavior ?

It's not really problematic, since there is a work around, but IMO this should be documented (either in the proxy setup or in the patch note) as I consider this to be a BC break

**Comment by blattersturm** [comment]
Thanks, this has been added to the pending changelog entries for the next server 'recommended' bump. Similarly, the console variable is actually noted in the message printed to the server console, so I'm not sure if this needs to be documented anywhere else - if you hit this issue, you're directly shown how to remediate it, which is _way_ better than it being hidden in some changelog or documentation only.
